app/src/main/java/ch/epfl/sdp/blindly/profile_setup/ProfileBirthday.kt

```diff
@@ -33,16 +33,16 @@ class Profile3 : AppCompatActivity() {
         if (age < MAJORITY_AGE) {
             findViewById<TextView>(R.id.warning_p3).visibility = View.VISIBLE
         } else {
-            val intent = Intent(this, Profile4::class.java)
+            val intent = Intent(this, ProfileGender::class.java)
             startActivity(intent)
         }
     }
 
     @RequiresApi(Build.VERSION_CODES.O)
     fun getAge(year: Int, month: Int, dayOfMonth: Int): Int {
         return Period.between(
-            LocalDate.of(year, month, dayOfMonth),
-            LocalDate.now()
+                LocalDate.of(year, month, dayOfMonth),
+                LocalDate.now()
```

Comment:
Same comment (why 2 tabs ?).

Reply:
It's non configurable when you pass the argument to the line below it starts from there.